mava/configs/env/lbf.yaml

```diff
@@ -0,0 +1,11 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - lbf_scenario: Foraging-2s-8x8-2p-2f-coop # [Foraging-2s-8x8-2p-2f-coop, Foraging-8x8-2p-2f-coop, Foraging-2s-10x10-3p-3f, Foraging-10x10-3p-3f, Foraging-15x15-4p-3f, Foraging-15x15-4p-5f]
+
+env_name: LevelBasedForaging-v0
+task_name: Foraging-2s-8x8-2p-2f-coop
+# Choose whether to use the use_idividual_reward = True OR aggregate_rewards=True.
+use_idividual_reward: False # If True, the network uses the list of different rewards given.
+aggregate_rewards : True  # If True, aggregates rewards across agents using the specified function.
+aggregate_rewards_with: sum # The function for aggregating rewards ("sum" or "mean").
```

Comment:
NIT: I think we could name this `reward_aggregation_function` ?

Reply:
Agree!